src/module.ts

```diff
@@ -52,8 +49,7 @@ export default defineNuxtModule<NuxtI18nOptions>({
     name: NUXT_I18N_MODULE_ID,
     configKey: 'i18n',
     compatibility: {
-      nuxt: '>=3.0.0-rc.11',
-      bridge: false
+      nuxt: '>=4.0.0'
```

Comment:
```suggestion
      nuxt: '>=3.0.0-rc.11'
```

I think with the changes so far we're still compatible with Nuxt 3 too 🤔